src/utils/external/index.ts

```diff
@@ -111,3 +111,32 @@ export const getBlockExplorerLink = ({
 
 export const getNetworkRelaseLink = (version: ColonyVersion) =>
   `${NETWORK_RELEASES}/${releaseMap[version]}`;
+
+// Add support for Beamer (getbeamer.com) in app feature announcements
+export const getBeamerId = process.env.BEAMER_PRODUCT_ID || false;
```

Comment:
This shouldn't live in utils, since it's not a util. This logic should just be wired in inside `index`, expecially since it's not being reused